### ISSUE TITLE
Double @Deprecated annotation

### ISSUE THREAD

**Issue by kassim** [apollographql/apollo-android#917]
I'm seeing deprecated values sometimes come in with double `@Deprecated` annotations in the generated code, which breaks the build

Unfortunately I can't give away all my code/schema, but this is the generated code, followed by the section of the schema that defines the deprecated value.

I'm not sure why the `accessToken` string isn't marked twice but the `expiresAt` is

```
    public AuthenticateUserWithFacebook(@Nonnull String __typename,
        @Nonnull @Deprecated String accessToken, @Deprecated @Deprecated long expiresAt,
        boolean isNew, @Nonnull RefreshToken refreshToken) {
      this.__typename = Utils.checkNotNull(__typename, "__typename == null");
      this.accessToken = Utils.checkNotNull(accessToken, "accessToken == null");
      this.expiresAt = expiresAt;
      this.isNew = isNew;
      this.refreshToken = Utils.checkNotNull(refreshToken, "refreshToken == null");
    }

...

    /**
     * Unix epoch time at which the token will expire. Uses UTC timezone.
     * @deprecated Use the expiresAt field inside the apiAccessToken value instead.
     */
    public @Deprecated @Deprecated long expiresAt() {
      return this.expiresAt;
    }

```

```
            {
          "kind": "OBJECT",
          "name": "AuthenticateFacebookUserPayload",
          "description": null,
          "fields": [
            {
              "name": "accessToken",
              "description": "Access token to authenticate further requests with.",
              "args": [],
              "type": {
                "kind": "NON_NULL",
                "name": null,
                "ofType": {
                  "kind": "SCALAR",
                  "name": "String",
                  "ofType": null
                }
              },
              "isDeprecated": true,
              "deprecationReason": "Use the tokenString field inside the apiAccessToken value instead."
            },
            {
              "name": "expiresAt",
              "description": "Unix epoch time at which the token will expire. Uses UTC timezone.",
              "args": [],
              "type": {
                "kind": "NON_NULL",
                "name": null,
                "ofType": {
                  "kind": "SCALAR",
                  "name": "Int",
                  "ofType": null
                }
              },
              "isDeprecated": true,
              "deprecationReason": "Use the expiresAt field inside the apiAccessToken value instead."
            },
```

**Comment by sav007** [comment]
What version are you using?

**Comment by kassim** [comment]
`0.4.4`

**Comment by sav007** [comment]
I think it was fixed in 0.5.0 version via https://github.com/apollographql/apollo-android/pull/863
If it's still reproducible feel free to reopen this issue.
